Correct AudioWorkletNode processorerror event handler

1valdis opened the pull request:
Rename `processorerror_event` to `onprocessorerror` because that's how the property is called on the AudioWorkletNode interface.
- [Link to the current spec](https://webaudio.github.io/web-audio-api/#audioworkletnode)

Maybe I should add separate `processorerror event`, so that we have `onprocessorerror` as an event handler property on AudioWorkletNode, and `processorerror event` as an event itself? Similarly to how it's done on OfflineAudioContext with `complete` event.

chrisdavidmills commented:
> Maybe I should add separate processorerror event, so that we have onprocessorerror as an event handler property on AudioWorkletNode, and processorerror event as an event itself? Similarly to how it's done on OfflineAudioContext with complete event.

Yes, you should do that. We recently changed the way we document events so that there is an event reference page for the event itself, which talks about the different ways to respond to the event (e.g. `on...` and `addEventListener`), and a separate event page for the `on...` property, if one exists for that event (not always the case, but usually). And the browser-compat-data needs to have a separate entry for each (which again, is often the same, but not always).

1valdis commented:
Hi @ddbeck, I've added the event. Also I've changed the ordering to alphabetical because as I can see on the other pages it is alphabetical. Something else?
